apollo.sh

```diff
@@ -698,6 +698,7 @@ function main() {
   check_esd_files
 
   DEFINES="--define ARCH=${MACHINE_ARCH} --define CAN_CARD=${CAN_CARD} --cxxopt=-DUSE_ESD_CAN=${USE_ESD_CAN}"
+  DEFINES="${DEFINES} --experimental_multi_threaded_digest"
```

Comment:
Thanks for the contribution!
1. Could you add some comment to explain this flag?
2. Have you tried and evaluated the speeding up?

Reply:
I mean, add comment in code. It could be similar to the pull-request explanation above.

Reply:
1. I pushed a new commit. If you want, I can squash it.
2. The travis CI build times are noisy. The best benchmarks can be best seen on an already built project *without* changes. My processor runs at 4 Ghz. It has 4 physical cores and 4 virtual cores. After I run `docker/scripts/dev_start.sh` on 6d159867dab292373cdd272655b6d5ee6fba8e4a, it takes 146 seconds to run `apollo.sh build`. After I ran `docker/scripts/dev_start.sh` on my patch, it took 55 seconds `apollo.sh build`.